fire/api/model/tidsserier.py

```diff
@@ -0,0 +1,80 @@
+from __future__ import annotations
```

Comment:
Stadig relevant?

Reply:
```suggestion
```

Reply:
Ja, beklager, men som v talte om, så kom knappen med mulighed for at lave ændringer fra min side ikke frem, så de her potentielle rettelser bliver lidt bøvlede at lave. Beklager.